### ISSUE TITLE
Domain name is not available

### ISSUE THREAD

**Issue by angoca** [dotevo/osm24#56]
I tried to access to your website but it is not longer available.

**Comment by dotevo** [comment]
Still works here http://osmapa.pl/osm24/

**Comment by naoliv** [comment]
Do anybody knows if there is an online instance of http://unexpired.osm24.eu/ somewhere else, please?

**Comment by ypid** [comment]
@dotevo Thanks for your work! Nice to hear that the site has not vanished :) When it is just an issue with the domain (domains a kind of expensive for my taste) then you should be able to get a nice subdomain like osm24.openstreetmap.de by creating your project here: https://wiki.openstreetmap.org/wiki/FOSSGIS/Server/Projects and then get in touch with the admins. They should be able to figure something out. Maybe even a CNAME to your server so that you can host the site when you prefer that.

**Comment by dotevo** [comment]
> Do anybody knows if there is an online instance of http://unexpired.osm24.eu/ somewhere else, please?

No. Nobody was interested (3 uniq entries per month) so it does not exist any more.

@ypid Works fine here: http://osmapa.pl/osm24/ but I do not have an access to source code etc.
Not important because traffic is very low maybe 1-2 person per day.

**Comment by ypid** [comment]
> Works fine here: http://osmapa.pl/osm24/

Works for me then :)
